src/main/kotlin/nexters/admin/domain/user/member/Member.kt

```diff
@@ -32,4 +33,15 @@ class Member(
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     var id: Long = 0L
+
+    fun validatePassword(password: Password) {
+        if (this.password.value != password.value) {
+            throw UnauthenticatedException.loginFail()
+        }
+    }
+
+    fun updatePassword(password: Password) {
+        this.password = password
+        this.isInitPassword = false
```

Comment:
초기화를 한번 진행했는지 물어보는 로직이니까 `true`로 바껴야되지 않나요?
Member에서 `isInitPassword`의 디폴트 값이 `false`로 지정돼있는 이유기도 하다고 생각해요.
제가 도메인 로직 이해를 잘못하고 있는건지 궁금합니다.
```suggestion
        this.isInitPassword = true
```

Reply:
아 저는 `isInitPassword`을 `isInitialPassword`, 즉 1234처럼 전화번호를 활용한 `초기 비밀번호 상태이냐 아니냐`로 이해했어요.
그래서 처음에 저장될 때는 `true`고 비밀번호 수정시, `false`가 된다고 이해했던거고요.

개인적으로 조금 네이밍이 헷갈리는데 내일 좀 얘기해볼까요?

Reply:
음 저는 isInitPassword -> 초기화가 진행이 됐는지 여부라서 비밀번호 수정되면 true로 바뀌는 게 나아보이는데 어떻게 생각하시나요?
만약 false로 할 거라면 프로퍼티의 디폴트값은 true로 바꿔주어야 할 것 같습니다.

Reply:
false => true로 수정하되, 초기화 진행 여부를 명시하기 위해 `hasChangedPassword`로 네이밍을 수정하는걸로 오프라인에서 이야기 완료!